pkg/arvo/lib/pill.hoon

```diff
@@ -19,7 +19,7 @@
 ::
 +$  dawn-event
   $:  =seed:able:jael
-      spon=[=ship point:azimuth]
+      spon=(list [=ship point:azimuth])
```

Comment:
Nitpick: it'd probably be better for `+unix-event` to use `$>(%dawn task:able:jael)`

Reply:
`+dawn` in lib/ph/azimuth produces a raw `+dawn-event`, and it's ugly to require it to add a tag for no reason.  I'll add +dawn-event to `able:jael` and use that everywhere.

Reply:
Ah, that makes sense. Sounds good.